### PR TITLE
Emigrate spinup to new template

### PR DESCRIPTION
**Pull request by sunnivin**
Adapted from the [old repository](https://github.com/sunnivin/rtd-github-pages/issues/7)

### CODE REVIEW

**Comment by sunnivin** [comment]
Remember to check that you got everything you mentioned @kjetilaas

File: content/spinup.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+    [~/HOME]$ projects
+
+after logging to a remote HPC-computer. This will return something that looks like this
+
+    nn2806k
+    nn1000k
+
+In the example above, two projects can be used (nn2806k and nn1000k).
+Then make sure you choose the right project when running CTSM.
```
Comment (ziu1986) [inline content/spinup.md:15]: Should we be more specific?
Which is the right project? 
In the example, nn2* is the "general" MetOS project. nn1* was the master student project on able - I don't know how this is handled now. 
> The right project is the one that your account accounts on (the one that you wrote into the form when you applied for computing time) -> `cost`

**Review by ziu1986 — APPROVED** [review]
Looks OK.

File: content/spinup.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
+# Setup CTSM
+
+For running CTSM on the Norwegian cluster you will be affilicated with a project account. 
+
+To check which project you are affiliated with can run by typing
+
+    [~/HOME]$ projects
```
Comment (emiliano-gelati) [inline content/spinup.md:7]: If `HOME` is the home directory environment variable, then should we have `~/$HOME`?

(related to https://github.com/MetOs-UiO/CTSM-Norway-Documentation/issues/3)

**Review by emiliano-gelati — APPROVED** [review]

File: content/spinup.md (hunk 2 above)
Comment (sunnivin) [reply content/spinup.md:7]: Related to #7 .

Let's do a total review on the point when the total version 1 of the documentation is ready.

**Comment by sunnivin** [comment]
Since 2 persons have approved my changes I will do the merge. 

If @kjetilaas finds out that something is missing, please file a PR :)

**Pull request merged**